bowtie/schemas/public-implementations.json

```diff
@@ -0,0 +1,77 @@
+{
+  "title": "Public Implementations",
+  "description": "A Public API providing metadata about implementations of JSON Schema supported by Bowtie",
+
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+
+  "$id": "tag:bowtie.report,2024:models:public-implementations",
+
+  "type": "object",
+  "propertyNames": {
+    "description": "A URL where the implementation's source code is hosted",
+
+    "type": "string",
+    "format": "uri"
+  },
+  "additionalProperties": {
+    "type": "object",
+    "required": ["name", "language", "dialects", "homepage", "issues"],
+    "properties": {
+      "name": { "type": "string" },
+      "language": {
+        "description": "The implementation language (e.g. C++, Python, etc.)",
+
+        "type": "string",
+        "pattern": "^[a-z][a-z0-9-+_]*$"
+      },
+      "dialects": {
+        "description": "A list of JSON Schema dialects (URIs) which the implementation understands. When running test cases, this list will be consulted before sending them to the implementation (and any unsupported dialects will be skipped).",
```

Comment:
The end of this description isn't relevant to this use case (there's no running of test cases as far as json-schema-org is concerned).